src/pages/myGeotab/guides/codeBase/usingInDotnet.tsx

```diff
@@ -0,0 +1,164 @@
+    servers, automatically serializing/deserializing JSON, and providing definitions for Checkmate object classes.
+    <br></br>
+    Please remember to regularly check for Geotab.Checkmate.ObjectModel nuget package updates. Ideally, your integration should use the same Geotab.Checkmate.ObjectModel nuget package version
+    as the one that your database is on.
+    <InformationalBox>
+        <p>Quick start in <a href="../../api/clients">{/*TODO: fix link*/}API Clients</a></p>
+    </InformationalBox>
+</div>
+
+const apiClass: ReactNode = <div>
+    <h2>Step 1: Initialize & Authentication</h2>
+    <p>In order to have access to the API class, add the following to the file's includes section:</p>
+    <CodeSample
+        language="csharp"
+        code={
+            `using Geotab.Checkmate;
+using Geotab.Checkmate.ObjectModel;`
+        }
+    ></CodeSample>
+    <p>An instance of API can now be constructed to be used in the code. For the most basic use case, all the data that is needed is user credentials and a database name:</p>
+    <CodeSample
+        language="csharp"
+        code={
+            `var api = new API(userName, password, null, databaseName, server);`
+        }
+    ></CodeSample>
+    <p>At this point there has not been any communication with Geotab's servers. In order to make calls to Geotab's API, an authentication call must be made:</p>
+    <CodeSample
+        language="csharp"
+        code={
+            `await api.AuthenticateAsync();`
+        }
+    ></CodeSample>
+    <p>When the call is made to Geotab’s servers to authenticate, a token is returned for security. Behind the scenes, the Authenticate call makes a JSON-RPC request to Geotab's
+        "Authenticate" method. The resulting security token and server information are stored in order to make further calls to the API.</p>
+    <InformationalBox>
+        <p>For more information regarding authentication, please review the {/*TODO: fix link*/}<a href="../concepts/#authentication">Authentication</a> documentation.</p>
+    </InformationalBox>
+    <h2>Step 2: Making Calls</h2>
+    <p >When authenticated, calls are made to the API by invoking the <code className="small-code-sample">{`Call`}</code> method of the API class.</p>
+    <p>The example below illustrates how to make a generic call to get all devices in the system.</p>
+    <code className="small-code-sample">{`List<Device> devices = await api.CallAsync<List<Device>>("Get", typeof(Device));`}</code>
+    <p>In the example below it is shown how to delete a device using the generic "Remove" method. Notice that it is not required to send all of the device's information to remove it,
+        the device's id is sufficient:</p>
+    <CodeSample
+        language="csharp"
+        code={
+            `await api.CallAsync<object>("Remove", typeof(Device), new {
+    entity = new Device {
+        Id = "b1a34"
+    }
+});`
+        }
+    ></CodeSample>
```

Comment:
This can be self-closing since there is no content inside the tags.
```suggestion
    />
```